packages/contracts/package.json

```diff
@@ -38,13 +38,13 @@
     "lint:check": "yarn run lint:typescript",
     "lint:typescript": "tslint --format stylish --project .",
     "clean": "rm -rf ./dist ./artifacts ./artifacts-ovm ./cache ./cache-ovm ./tsconfig.build.tsbuildinfo",
-    "deploy": "./bin/deploy.ts && yarn generate-markdown",
+    "deploy": "ts-node \"./bin/deploy.ts\" && yarn generate-markdown",
```

Comment:
The big quotes are required?

Reply:
Ermmmm they definitely *used* to be required when I was working on windows a year ago. Maybe things have changed. Just being safe. We might want to do a full review of windows support later.